content/_data/changelogs/lts.yml

```diff
@@ -11672,6 +11672,469 @@
+    references:
+      - pull: 9387
+      - issue: 73129
+      - url: /security/advisory/2021-10-06/#SECURITY-2481
+        title: Path traversal vulnerability on Windows - SECURITY-2481
+    message: |-
+      Remove Windows path traversal vulnerability escape hatch that was provided with the SECURITY-2481 fix..
+  - type: major bug
+    category: bug
+    pull: 9428
+    issue: 73381
+    authors:
+      - basil
+    pr_title: "[JENKINS-73381] Downloading tar.gz artifacts in Firefox is broken"
+    message: |-
+      Fix download of <code>.tar.gz</code> artifacts in Firefox.
+  - type: rfe
+    category: rfe
+    authors:
+      - janfaracik
+      - mawinter69
+      - timja
+    pr_title: "Refine content and appearance of the user account screen"
+    references:
+      - pull: 9521
+      - pull: 9707
+      - pull: 9461
+      - pull: 9411
+      - pull: 9393
+      - pull: 9381
+    message: |-
+      Enhancements and refinements for several pages' appearance in Jenkins.
+  - type: rfe
+    category: rfe
+    authors:
+      - zbynek
+      - janfaracik
+    pr_title: "Refine form controls appearance"
+    references:
+      - pull: 9453
+      - pull: 9380
+      - pull: 9365
+      - pull: 9395
+    message: |-
+      Refinements and modernizations for parts of Jenkins UI.
+  - type: rfe
+    category: rfe
+    pull: 7268
+    issue: 69869
+    authors:
+      - Wadeck
+      - timja
+    pr_title: "[JENKINS-69869] Categorize the user properties"
+    message: |-
+      User properties are now categorized in different pages.
+  - type: rfe
+    category: rfe
+    pull: 9148
+    authors:
+      - janfaracik
+    pr_title: "Rewrite the build history widget"
+    message: |-
+      Update the design of the build history widget.
+  - type: rfe
+    category: rfe
+    pull: 9724
+    authors:
+      - janfaracik
+    pr_title: "Use Notice component for views lacking jobs"
+    message: |-
+      Use Notice component for views lacking jobs.
+  - type: rfe
+    category: rfe
+    pull: 9648
+    issue: 73669
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-73669] don't change unrelated checkboxes in rowSelectionCont…"
+    message: |-
+      Don't change unrelated checkboxes in <code>rowSelectionController</code>.
+  - type: rfe
+    category: rfe
+    pull: 9591
+    authors:
+      - jglick
+    pr_title: "Friendlier handling of DeploymentHandshakeException from CLI in -webSocket mode"
+    message: |-
+      Better display HTTP handshake errors (such as authentication issues) from the CLI in <code>-webSocket</code> mode.
+  - type: rfe
+    category: rfe
+    pull: 9665
+    authors:
+      - Vlatombe
+    pr_title: "Add -webSocket option by default when creating an inbound agent"
+    message: |-
+      Use websocket in the inbound agent command line sample.
+  - type: rfe
+    category: rfe
+    pull: 9150
+    authors:
+      - krisstern
+    pr_title: "Add doCheckDisplayNameOrNull to jenkins core"
```

Comment:
I think this should be removed from the changelog.  I don't know what users will do with this information.  Since the changelog is focused on users rather than core maintainers or plugin maintainers, I think it is safe to remove this entry.

Note that this suggestion should not be applied because it does not propose to delete all the lines of the entry.  I included the suggestion to hint what needs to be done, knowing that the suggestion is incomplete.
```suggestion
```

Reply:
Thanks for the insight on this @MarkEWaite, I've adjusted things accordingly.